### PR TITLE
Add Xiaomi Aqara Door Lock

### PR DESCRIPTION
**Pull request by syssi**
**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#14419

### CODE REVIEW

**Review by SchumyHao — CHANGES_REQUESTED** [review]
Xiaomi Aqara Lock **CANNOT** unlock from Homeassistant, but we can use it's attribute to trigger automation.

`changed_by`: Show an unique ID number represent to last unlock method. 
`verified_wrong_times`: If someone tried to unlock this lock but failed more than 3 times, this attribute will record attempt times. This attribute will clear if unlock successfully.

**Comment by syssi** [comment]
Thanks for your input. I will add it to the PR as soon as possible.

File: source/_components/lock.xiaomi_aqara.markdown [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+title: "Xiaomi Aqara Lock"
+description: "Instructions on how to setup the Xiaomi Aqara Lock within Home Assistant."
+date: 2018-05-17 20:24
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: xiaomi.png
+ha_category: Lock
+ha_release: "0.70"
```
Comment (MartinHjelmare) [inline source/_components/lock.xiaomi_aqara.markdown]: 0.71

File: source/_components/lock.xiaomi_aqara.markdown [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ha_iot_class: "Local Push"
+---
+
+
+The `xiaomi aqara` lock platform allows you to get data (`changed_by` property, `verified_wrong_times` attribute) from your [Xiaomi](http://www.mi.com/en/) Aqara locks.
+A Aqara lock cannot be controlled by Home Assistant. The property `changed_by` provides the user/key ID of the last successful unlock.
+
+If someone tried to unlock the device but failed more than 3 times the `verified_wrong_times` attribute will be incremented. The counter resets on a successful unlock.
```
Comment (MartinHjelmare) [inline source/_components/lock.xiaomi_aqara.markdown]: If someone tries to unlock the device but fails more than 3 times...

File: source/_components/lock.xiaomi_aqara.markdown [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+The requirement is that you have setup [Xiaomi aqara](/components/xiaomi_aqara/).
```
Comment (MartinHjelmare) [inline source/_components/lock.xiaomi_aqara.markdown]: The requirement to use this platform is that you have set up the Xiaomi Aqara component.

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**